Generator.Equals/ContainingTypesBuilder.cs

```diff
@@ -81,8 +46,8 @@ public static string Build(ISymbol symbol, Action<IndentedTextWriter> content, b
                         var x => throw new ArgumentOutOfRangeException($"Syntax kind {x} not supported")
                     };
 
-                    var typeName = s.ToDisplayString(SymbolDisplayFormat.MinimallyQualifiedFormat);
-                    writer.WriteLine($"partial {keyword} {typeName}");
+                    // var typeName = parentSymbol.ToDisplayString(SymbolDisplayFormat.MinimallyQualifiedFormat);
```

Comment:
Feels more like the huge pile of stuff. You know the thing you are looking for is somewhere in the history, but you cant find it because things like the file doesnt exist anymore or you dont know *exactly* what you are looking for 😅 

Anyway,  this commented code was unnecessary anyway 😅